how-to-guides/e2e-tracking/notebooks/e2e_tracking.ipynb

```diff
@@ -69,10 +69,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Run this if you get a `RuntimeError: main thread is not in main loop` error\n",
-    "import matplotlib\n",
+    "# To fix the random RuntimeError: main thread is not in main loop error in Windows\n",
```

Comment:
```suggestion
    "# To fix the random `RuntimeError: main thread is not in main loop error` in Windows\n",
```
Ticks made it more readable 🤔

Reply:
It is a comment in a code cell, so it won't be rendered as inline code even when enclosed within ticks

Reply:
It doesn't matter that it's not rendered, it's about separating `in Windows` from the error message.

Reply:
```suggestion
    "# To fix the `RuntimeError: main thread is not in main loop` error in Windows\n",
```